index.js

```diff
@@ -396,7 +397,12 @@ require('util').inherits(TChannelConnection, require('events').EventEmitter);
 
 TChannelConnection.prototype.onParserError = function onParserError(err) {
     var self = this;
-    self.channel.logger.error('tchannel parse error', {
+    var mess = 'tchannel parse error';
+    if (err.buffer) {
+        mess += '\n' + hexer(err.buffer);
```

Comment:
consider adding a second `logger.debug()` message for the buffer.

Remember that errors go to sentry.